.github/workflows/ci.yml

```diff
@@ -104,14 +110,7 @@ jobs:
           printf 'outputs.tar should be a file\n'
           test -f "${{ steps.upload-rust-binary-action.outputs.tar }}"
 
-          printf 'outputs.sha256 should be a file\n'
-          test -f "${{ steps.upload-rust-binary-action.outputs.sha256 }}"
-
-          printf 'outputs.sha512 should be a file\n'
-          test -f "${{ steps.upload-rust-binary-action.outputs.sha512 }}"
-
-          printf 'outputs.sha1 should be a file\n'
-          test -f "${{ steps.upload-rust-binary-action.outputs.sha1 }}"
-
-          printf 'outputs.md5 should be a file\n'
-          test -f "${{ steps.upload-rust-binary-action.outputs.md5 }}"
+          for hash_type in ${ ${{ matrix.checksums || 'b2,sha256,sha512,sha1,md5' }}//,/ }; do
```

Comment:
https://github.com/taiki-e/upload-rust-binary-action/actions/runs/11762802533

> The workflow is not valid. .github/workflows/ci.yml (Line: 103, Col: 14): Unexpected symbol: '$hash_type'. Located at position 41 within expression: steps.upload-rust-binary-action.outputs.$hash_type

I think you have to assign `${{ matrix.checksums || 'b2,sha256,sha512,sha1,md5' }}` to a variable once and then split it. (Or use a while loop like a code handles INPUT_CHECKSUM.)

Reply:
@taiki-e I'm having a hard time finding the proper syntax for variable substitution in a GitHub Action expression. I'll try again later, in the meantime let me know if you have any idea.

The following doesn't seem to work:

```bash
test -f "${{ steps.upload-rust-binary-action.outputs.$hash_type }}"
```

Reply:
Resolved now